src/govuk/components/summary-list/_index.scss

```diff
@@ -166,4 +166,72 @@
       }
     }
   }
+
+  // Additional block for the summary card
+  //
+  // We ship this with the summary list instead of making it it's own component
+  // because it's only interacted with via the summary list and it avoids awkwardness
+  // with trying to document a "private" component that's controlled via the
+  // component it's wrapping
+  .govuk-summary-card {
+    @include govuk-responsive-margin(6, "bottom");
+    border: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-summary-card__title-wrapper {
+    display: flex;
```

Comment:
I like how we're using flexbox here, but it does mean that you can end up with the summary list actions floated to the right with the card actions on the left. I personally don't see this is an issue but wanted to flag it to make sure we're happy:

![Screenshot 2023-01-11 at 09 14 44](https://user-images.githubusercontent.com/121939/211765827-4c109154-8269-4987-b314-bd7660045005.png)

Reply:
I suppose ideally it would all break at the same point so that the actions were placed consistently. Is that possible given the flexbox thingy above breaks in relation to the width of the content rather than any consistent measure?

Reply:
If there's anything that can make them break at the same point that would be really neat but I don't feel it blocks publication if not :)